aptos-move/parallel-executor/src/proptest_types/tests.rs

```diff
@@ -33,12 +42,14 @@ where
         *transactions.get_mut(i.index(length)).unwrap() = Transaction::SkipRest;
     }
 
-    let baseline = ExpectedOutput::generate_baseline(&transactions);
-
     let output = ParallelTransactionExecutor::<Transaction<K, V>, Task<K, V>>::new()
-        .execute_transactions_parallel((), transactions);
+        .execute_transactions_parallel((), transactions.clone());
+
+    let baseline = ExpectedOutput::generate_baseline(&transactions);
 
-    baseline.check_output(&output)
+    iter::repeat_with(|| baseline.check_output(&output))
```

Comment:
what does this mean? checks `num_repeat` times output? I assume you mean to run the execution `num_repeat` times?

Reply:
damn, good catch! I was experimenting with something and uploaded a stupid version.